sml_builder/__init__.py

```diff
@@ -26,7 +26,10 @@
 
 @app.route("/")
 def index():
-    return render_template("index.html")
+    return render_template(
+        "index.html",
+        feature_1_active=sml_builder.utils.get_feature_config("FEATURE_1_ACTIVE"),
```

Comment:
Again, drop active - if you want the local variable to be declarative you can call it placeholder_feature_enabled = sml_builder.utils.get_feature_config("PLACEHOLDER_FEATURE")